diff -Ncr a/instrumentation/syscalls/headers/arm-32-syscalls-2.6.38_integers_override.h b/instrumentation/syscalls/headers/arm-32-syscalls-2.6.38_integers_override.h
*** a/instrumentation/syscalls/headers/arm-32-syscalls-2.6.38_integers_override.h	2013-07-12 05:29:25.000000000 +0900
--- b/instrumentation/syscalls/headers/arm-32-syscalls-2.6.38_integers_override.h	2013-12-11 23:00:01.605106000 +0900
***************
*** 2,8 ****
  
  #define OVERRIDE_TABLE_32_sys_arm_fadvise64_64
  #define OVERRIDE_TABLE_32_sys_sync_file_range2
- #define OVERRIDE_TABLE_32_sys_set_tls
  
  #ifndef CREATE_SYSCALL_TABLE
  
--- 2,7 ----
***************
*** 38,67 ****
  	TP_printk()
  )
  
- SC_TRACE_EVENT(sys_set_tls,
- 	TP_PROTO(unsigned int tid, unsigned long tls),
- 	TP_ARGS(tid, tls),
- 	TP_STRUCT__entry(
- 		__field(unsigned int, tid)
- 		__field_hex(unsigned int, tls)),
- 	TP_fast_assign(
- 		tp_assign(tid, tid)
- 		tp_assign(tls, tls)),
- 	TP_printk()
- )
- 
  #else	/* CREATE_SYSCALL_TABLE */
  
  #define OVVERRIDE_TABLE_32_sys_mmap
! TRACE_SYSCALL_TABLE(sys_mmap, sys_mmap, 90, 6) 
  
  #define OVERRIDE_TABLE_32_sys_arm_fadvise64_64
  TRACE_SYSCALL_TABLE(sys_arm_fadvise64_64, sys_arm_fadvise64_64, 270, 4)
  #define OVERRIDE_TABLE_32_sys_sync_file_range2
  TRACE_SYSCALL_TABLE(sys_sync_file_range2, sys_sync_file_range2, 341, 4)
- #define OVERRIDE_TABLE_32_sys_set_tls
- TRACE_SYSCALL_TABLE(sys_set_tls, sys_set_tls, 0xf0005, 2)
- 
  
  #endif /* CREATE_SYSCALL_TABLE */
  
--- 37,51 ----
  	TP_printk()
  )
  
  #else	/* CREATE_SYSCALL_TABLE */
  
  #define OVVERRIDE_TABLE_32_sys_mmap
! TRACE_SYSCALL_TABLE(sys_mmap, sys_mmap, 90, 6)
  
  #define OVERRIDE_TABLE_32_sys_arm_fadvise64_64
  TRACE_SYSCALL_TABLE(sys_arm_fadvise64_64, sys_arm_fadvise64_64, 270, 4)
  #define OVERRIDE_TABLE_32_sys_sync_file_range2
  TRACE_SYSCALL_TABLE(sys_sync_file_range2, sys_sync_file_range2, 341, 4)
  
  #endif /* CREATE_SYSCALL_TABLE */
  
