diff -crN a/configure.ac b/configure.ac
*** a/configure.ac	2013-04-01 23:31:20.343008000 +0900
--- b/configure.ac	2013-11-29 00:40:07.017478455 +0900
***************
*** 158,174 ****
  done
  
  # Check for proper Qt version.
! AC_CACHE_CHECK([for Qt library version >= 4.2],
!   ac_cv_qtversion, [
!   AC_TRY_LINK([#include "QtCore/qglobal.h"], [
! #if QT_VERSION < 0x040200
! #error Qt library 4.2 or greater required.
! #endif
!     ], ac_cv_qtversion="yes", [
!       echo "no; Qt 4.2 or greater is required"
!       exit
!     ])
! ])
  
  # A common error message:
  ac_errmsg="not found in current PATH. Maybe QT development environment isn't available (qt-devel)."
--- 158,164 ----
  done
  
  # Check for proper Qt version.
! ac_cv_qtversion="yes"
  
  # A common error message:
  ac_errmsg="not found in current PATH. Maybe QT development environment isn't available (qt-devel)."
